### DISCUSSION TITLE
Double authentification

### DISCUSSION THREAD

**Discussion by AteneaV** [community/community#52089] [New to GitHub]
When I wanted to have the access to my account, I wrote my user and my password and I passed the first authentication, after I saw a message for the second authentication, at this time I can not do that because I saw an old email where I worked in my last job and I don't have the possibility to change that, finally,  I try with my personal email, not my user, the system show me a message for to change my password, and I access.
The other when you look how you can contact you, always you have the robot and there is not a button for close you use the line of the site and you can come back.

**Comment by BREN0sx** [comment]
To resolve this issue, you will need to update your 2FA settings on GitHub. You can do this by logging in to your account with your personal email address, then navigating to your account settings and selecting the "Security" tab.

From there, you should be able to update your 2FA settings to use a different authentication method, such as a security key or a mobile app. You can also remove any old email addresses from your account to prevent this issue from happening again in the future.

Regarding the contact options for GitHub support, you can find a link to their contact form at the bottom of their website's homepage. From there, you can select the category of your issue and submit a support request to their team.
